redbot/cogs/audio/sql_statements.py

```diff
@@ -306,30 +387,59 @@
     track_info = excluded.track_info,
     last_updated = excluded.last_updated;
 """
-SPOTIFY_UPDATE = """
+SPOTIFY_UPDATE: Final[
+    str
+] = """
 UPDATE spotify
 SET last_fetched=:last_fetched
 WHERE uri=:uri;
 """
-SPOTIFY_QUERY = """
+SPOTIFY_QUERY: Final[
+    str
+] = """
 SELECT track_info, last_updated
 FROM spotify
 WHERE
     uri=:uri
     AND last_updated > :maxage
 LIMIT 1;
 """
-SPOTIFY_DELETE_OLD_ENTRIES = """
+SPOTIFY_QUERY_ALL: Final[
+    str
+] = """
+SELECT track_info, last_updated
+FROM spotify
+"""
+SPOTIFY_DELETE_OLD_ENTRIES: Final[
+    str
+] = """
 DELETE FROM spotify
 WHERE
-    last_updated < :maxage;
+    last_updated < :maxage
+    LIMIT 100
```

Comment:
`LIMIT` shouldn't be specified in this query.
```suggestion
    last_updated < :maxage
```